src/helpers/traveltimesHelper.ts

```diff
@@ -19,11 +21,31 @@ export async function lineStringsFromTraveltimes(
       traveltime.end
     )
 
-    const lineString = new LineString([
-      startTrambusStop?.getGeometry()?.getCoordinates(),
-      endTrambusStop?.getGeometry()?.getCoordinates(),
-    ])
-    return lineString
+    if (is3D) {
+      const cesiumMap = rennesApp.get3DMap()
```

Comment:
Maybe add a comment to explain the code below
For example : "Modify coordinates by adding an altitude" , something like that